Mikado-Website/index.html

```diff
@@ -0,0 +1,176 @@
+
+  <!-- Libraries CSS Files -->
+  <link href="lib/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+  <link href="lib/fancybox/fancybox.css" rel="stylesheet">
+
+  <!-- Main Stylesheet File -->
+  <link href="css/style.css" rel="stylesheet">
+
+  <link rel="prefetch" href="img/zoom.png">
+
+
+</head>
+
+<body data-spy="scroll" data-offset="58" data-target="#navigation">
+
+  <!-- Fixed navbar -->
+  <div id="navigation" class="navbar navbar-fixed-top">
+    <div class="container">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+          <i class="fa fa-bars"></i>
+        </button>
+      </div>
+      <!--/.nav-collapse -->
+    </div>
+  </div>
+
+  <!-- === MAIN Background === -->
+  <div class="slide story" id="intro" data-slide="1">
+    <div class="container">
+      <div id="home-row-1" class="row clearfix">
+        <div class="col-12">
+          <h1 class="font-semibold">MIKADO </h1>
+          <h4 class="font-thin"> <span class="font-semibold">Organize </span> your next adventure.</h4>
+          <br>
+          <br>
+        </div>
+        <!-- /col-12 -->
+      </div>
+      <!-- /row -->
+      <!-- /row -->
+    </div>
+    <!-- /container -->
+  </div>
+  <!-- /slide1 -->
+
+  <!-- === Slide 2 === -->
+  <div class="slide story" id="about-us" data-slide="2">
+    <div class="container">
+      <div class="row title-row">
+        <div class="col-12 font-thin"> Scroll left or right to preview Mikado<br> </div>
+      </div>
+      <!-- /row -->
+      <div class="row line-row">
+        <div class="hr">&nbsp;</div>
+      </div>
+      <!-- /row -->
+      <div class="row subtitle-row">
+      </div>
+        <center>
+            
+    <div class="slider">
+
+        
+<!-- Button Implementation that is currently super buggy (FIX: stop it from shifting the page up and down)
+        <a href="#slide-1">1</a>
+        <a href="#slide-2">2</a>
+        <a href="#slide-3">3</a>
+-->
+        
+        <div class="slides">
+
+            <div id="slide-1"><img src="img/Homescreen.png"></div>
+            <div id="slide-2"><img src="img/screen2.png"></div>
+            <div id="slide-3"><img src="img/screen3.png"></div>
+
+        </div>
+ 
+    
+    </div>
+            </center>
+      
+      <!-- /row -->
+    </div>
+    <!-- /container -->
+  </div>
+  <!-- /slide2 -->
+    
+
+  <!-- /slide3 -->
+
+  <!-- === Slide 5 === -->
+  <div class="slide story" id="clients" data-slide="5">
+    <div class="container">
+      <div class="row title-row">
+        <div class="col-12 font-thin"><span class="font-semibold">Sponsors</span> we’ve worked with</div>
+      </div>
+      <!-- /row -->
+      <div class="row line-row">
+        <div class="hr">&nbsp;</div>
+      </div>
+      <!-- /row -->
+      <div class="row subtitle-row">
+        <div class="col-12 font-light">Our Sponsors.</div>

```

Comment:
Either we can put our UBC launch pad sponsors here or just leave the sponsors section out